### ISSUE TITLE
Improve accessibility sitewide

### ISSUE THREAD

**Issue by evertiro** [FuelRats/fuelrats.com#426]
We already have one ticket related to accessibility for input groups, but we have a bigger problem - the white on red we're using in many places throughout the site fails WCAG compliance for text (works for UI components though). We need to do an accessibility audit and see what all needs to be fixed, and decide how to fix them.

**Comment by UncleClapton** [comment]
From our discussion on button colors.

> moving to #b94545 would probably work for small buttons, but #d65050 is a color so closely tied to our branding. gotta use it somewhere 😛
